docs/how-to/command-line-quick-start.md

```diff
@@ -1,178 +1,243 @@
+
+    ```
+    Hello and Welcome to IPFS!
+
+    ██╗██████╗ ███████╗███████╗
+    ██║██╔══██╗██╔════╝██╔════╝
+    ██║██████╔╝█████╗  ███████╗
+    ██║██╔═══╝ ██╔══╝  ╚════██║
+    ██║██║     ██║     ███████║
+    ╚═╝╚═╝     ╚═╝     ╚══════╝
+
+    If you see this, you have successfully installed
+    IPFS and are now interfacing with the ipfs merkledag!
+
+    -------------------------------------------------------
+    | Warning:                                              |
+    |   This is alpha software. use at your own discretion! |
+    |   Much is missing or lacking polish. There are bugs.  |
+    |   Not yet secure. Read the security notes for more.   |
+    -------------------------------------------------------
+
+    Check out some of the other files in this directory:
+
+      ./about
+      ./help
+      ./quick-start     <-- usage examples
+      ./readme          <-- this file
+      ./security-notes
+    ```
+
+1. The respository's `quick-start` directory shows other example commands to try. To display the contents of `quick-start, run:
+
+    ```bash
+    ipfs cat /ipfs/QmYwAPJzv5CZsnA625s3Xf2nemtYgPpHdWEz79ojWnPbdG/quick-start
+    ```
+
+   :::tip
+   There are a whole host of other configuration options you can set — see the [the full reference](https://github.com/ipfs/kubo/blob/master/docs/config.md) for more.
+   :::
 
 ## Take your node online
 
-Once you're ready to join your node to the public network, run the ipfs daemon in another terminal and wait for all three lines below to appear to know that your node is ready:
+Next, take your node online and interact with the IPFS network:
 
-```bash
-ipfs daemon
+1. Open another terminal window. 
 
-> Initializing daemon...
-> API server listening on /ip4/127.0.0.1/tcp/5001
-> Gateway server listening on /ip4/127.0.0.1/tcp/8080
-```
+2. Start the IPFS daemon in the new terminal window:
 
-Make a note of the TCP ports you receive. If they are different, use yours in the commands below.
+    ```bash
+    ipfs daemon
+    ```
 
-::: danger NEVER EXPOSE THE RPC API TO THE PUBLIC INTERNET
+   After a few moments, output like the following displays, and your node is ready:
 
-The API port provides admin-level access to your Kubo IPFS node.  See [RPC API v0 docs](../reference/kubo/rpc.md) for more information.
+    ```bash
+    > Initializing daemon...
+    > API server listening on /ip4/127.0.0.1/tcp/5001
+    > Gateway server listening on /ip4/127.0.0.1/tcp/8080
+    ```
 
-:::
+    Make a note of the TCP ports in the output. If they are different, use yours in the commands below.
 
-Now, switch back to your original terminal. If you’re connected to the network, you should be able to see the IPFS addresses of your peers when you run:
+    ::: danger NEVER EXPOSE THE RPC API TO THE PUBLIC INTERNET
 
-```bash
-ipfs swarm peers
+    The API port provides admin-level access to your Kubo IPFS node.  See [RPC API v0 docs](../reference/kubo/rpc.md) for more information.
```

Comment:
```suggestion
    The API port (by default `5001`) provides admin-level access to your Kubo IPFS node.  See [RPC API v0 docs](../reference/kubo/rpc.md) for more information.
```

Reply:
> > Step 4 of https://bafybeidvobdv5i3pjscvsityfe6kzpkaqmdze6jyk5irmzydmaf7yex2ea.on.fleek.co/how-to/command-line-quick-start/#interact-with-the-node-using-the-web-console is not working for me - when I try to copy the file over to the MFS that was added in step 8 of https://bafybeidvobdv5i3pjscvsityfe6kzpkaqmdze6jyk5irmzydmaf7yex2ea.on.fleek.co/how-to/command-line-quick-start/#take-your-node-online using its hash, I see Error: argument "dest" is required.
> 
> The command should be `ipfs files cp /ipfs/QmabZ1pL9npKXJg8JGdMwQMJo2NCVy9yDVYjhiHK4LTJQH /meow.txt`
> 
> You can think of the `ipfs files cp` command as similar to `cp` with the difference that `/` is the root of MFS.

Cheers thanks @2color confirmed this on my local and updated the copy https://github.com/ipfs/ipfs-docs/pull/1444/files#diff-3663ece73a79ceea21ae21afd88f542259018e3453af373d9909e8466f96e8adR228-R232